compiler/dyno/util/linters/CMakeLists.txt

```diff
@@ -17,22 +17,19 @@
 
 add_executable(fieldsUsed "fieldsUsed.cpp")
 
+# This option is set from the call to CMake from compiler/dyno/Makefile.help
 option(DYNO_LINTERS_CLANG_CMAKE_PATH "Override the path to find a ClangConfig.cmake")
 
+# TODO: Maybe we shouldn't set this for CHPL_LLVM=none?
 if(DYNO_LINTERS_CLANG_CMAKE_PATH)
   # this finds the target clang-cpp and also finds an appropriate LLVM version
+  set(Clang_ROOT ${DYNO_LINTERS_CLANG_CMAKE_PATH})
```

Comment:
What is this for, I don't see it anywhere else in the PR.

Reply:
to force the call to `find_package(clang)` to locate the right one, the one at the path we passed in using the option `-DDYNO_LINTERS_CLANG_CMAKE_PATH`

Reply:
This may be a vestigial thought of mine related to the problems we had with `find_package(LLVM)` last night

Reply:
Ah, I see. `<package>_ROOT` is used by `find_package` then?

Reply:
yeah, the docs (https://cmake.org/cmake/help/latest/command/find_package.html#config-mode-search-procedure) state it's the first place it looks, if I am understanding correctly.